packages/vendure-plugin-popularity-scores/src/sort.service.ts

```diff
@@ -85,7 +83,7 @@ export class SortService implements OnModuleInit {
       .addGroupBy('product.id')
       .addOrderBy('count', 'DESC')
       .getRawMany();
-    const maxCount = groupedOrderItems?.[0]?.count;
+    const maxCount = groupedOrderLines?.[0]?.count;
```

Comment:
I think we need a modification for this to work properly:

In v1 we could count the orderItems, which is correct, but here (in v2) we are currently counting order lines. But order lines can have different quantities, so we should take the sum of all `orderlines.quantity` correct?

Or, am I missing something? (Tests are succeeding, but I am not sure if the tests cover this)

Reply:
Yeah you're right. I will look into it